backends/backend_cryptombssl.c

```diff
@@ -337,6 +345,8 @@ static int cryptomb_ssl_ecdsa_sigver(struct ecdsa_sigver_data *data, flags_t par
         data->sigver_success = 0;
     }
 
+out:
```

Comment:
Is this label used in CKNULL_LOG? If no, then looks like it is not used in this function

Reply:
Yep, it is. `CKNULL_LOG` checks for an error and goes to the end of the function by this label:

```
#define CKNULL_LOG(v, r, ...) {						\
	if (!v) {							\
		logger(LOGGER_ERR, __VA_ARGS__);			\
		ret = r;						\
		goto out;						\
	}								\
}
```